examples/bond/scripts/run/common/common.js

```diff
@@ -68,6 +68,209 @@ async function optInTx (deployer, managerAcc, lsig, assetIndex) {
   await executeTransaction(deployer, optInTx);
 }
```

Comment:
The `optInTx` doesn't require a "from" address to be a manager. In fact it's an address which pays for the tx fee. 
1. Let's rename `managerAcc` to `payerAcc`
2. Let's update the documentation saying that this creates a group tx where first transaction will pay the tx fees and second will do the opt-in.

Reply:
- This transaction requires this, this is because no one can opt-in issuer, dex lsigs to asa except manager.
- If everyone can do this lsig addresses will be spammed.

Reply:
Updating documentation

Reply:
right. thanks for updating docs.